root/static/scripts/common/utility/formatTrackLength.js

```diff
@@ -1,13 +1,17 @@
 /*
+ * @flow strict-local
  * Copyright (C) 2011 MetaBrainz Foundation
  *
  * This file is part of MusicBrainz, the open internet music database,
  * and is licensed under the GPL version 2, or (at your option) any
  * later version: http://www.gnu.org/licenses/gpl-2.0.txt
  */
 
-export default function formatTrackLength(milliseconds, placeholder='?:??') {
-    if (!milliseconds) {
+export default function formatTrackLength(
+    milliseconds?: number | null,
+    placeholder: string = '?:??',
+): string {
```

Comment:
The `?` seems mixed up here
```suggestion
export default function formatTrackLength(
    milliseconds: number | null,
    placeholder?: string = '?:??',
): string {
```